### PR TITLE
MCS-467: Dynamically Create and Show CSV, along with adding Admin Features

### PR DESCRIPTION
**Pull request by brianpippin**
Features:
Test two things first:
1.  You should NOT have an Administrator link in the User Drop Down on the Right side of the NavBar
2.  You should NOT have a "Create CSV" button on the Evaluation Status Page

Now run the script that is in mcs-ingest to create the initial admin users, you should be one of them.
1.  Now you should be able to see the Adminstrator Link and the Create CSV link
2. You should be able to open the Create CSV Modal and select if you want to create a Scene or Results CSV
3. The CSV automatically get uploaded to the evaluation-images bucket (example link:  https://resources.machinecommonsense.com/csv-db-files/Evaluation_3.75_Results.csv)
4. If you want to double check the files are being created (as I have created most at this point you can delete it from the bucket or check the time stamp in the bucket)
5. The links to download the CSV appear after a pre-flight test to make sure those links exist, if they exist they show up, if they 404 they do not show up.
6. There is now an Adminstrator page under the right nav drop down.  Here only admin users can add or remove people to be admin users.  Currently Admin users can only create CSV and modify the admin user list, but we could add more functionality in the future.

### CODE REVIEW

**Review by rartiss55 — APPROVED** [review]

File: analysis-ui/src/components/Account/adminPage.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import React, {useState} from 'react';
+import {withRouter} from 'react-router-dom';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import '../../css/adming-page.css';
```
Comment (rartiss55) [inline analysis-ui/src/components/Account/adminPage.jsx:5]: Typo in filename for admin-page.css

File: analysis-ui/src/components/App/index.jsx [hunk 2]
```diff
@@ -149,7 +149,12 @@ function Admin({newState, userLoginHandler}) {
         history.push("/login");
     }
 
-    return <AdminPage currentUser={newState.currentUser} updateUserHandler={userLoginHandler}/>
+    // Do not let users who aren't admins somehow go to the admin page
+    if(newState.currentUser !== null && newState.currentUser.admin === true) {
```
Comment (rartiss55) [inline analysis-ui/src/components/App/index.jsx:153]: Nice catch!

**Comment by rartiss55** [comment]
This looks great -- only noticed this on a second pass, if you use the admin page to modify users and leave the page/come back without doing a hard refresh, the lists don't stay updated (its probably just caching the previous getUsers query results from before the modification).

File: analysis-ui/src/components/Account/adminPage.jsx (hunk 1 above)
Comment (brianpippin) [reply analysis-ui/src/components/Account/adminPage.jsx:5]: Good catch, fixed the names.

**Comment by brianpippin** [comment]
> This looks great -- only noticed this on a second pass, if you use the admin page to modify users and leave the page/come back without doing a hard refresh, the lists don't stay updated (its probably just caching the previous getUsers query results from before the modification).

It's suppose to update the user as soon as you move it back and forth, but I will check to make sure the getUsers call isn't caching, will test this some more before merging.

**Pull request merged**